tests/qinfo/test_fisher.py

```diff
@@ -157,7 +157,7 @@ def qfunc(params):
             qml.RX(params[0], wires=0)
             qml.RX(params[1], wires=0)
             qml.CNOT(wires=(0, 1))
-            return qml.state()
+            return qml.probs()
```

Comment:
?

Reply:
this circuit is used with `quantum_fisher` and `(adjoint_)metric_tensor` below - they don't really care about the measurement process being used, but `dev_hard` has shots set. If you try to use the QNode, it will fail saying you can't return a state measurement with shots set. But we need to set shots to make sure it dispatches properly within PennyLane! chatted with Mudit and we felt pretty certain this was best. plus tests passed : )